midterm-2023-11-06/code/5100-nenad-saldziev/script.ts

```diff
@@ -0,0 +1,188 @@
+interface albums{ 
```

Comment:
Interfaces are mostly ok, just that `[string]` means something different from `string[]` which is the correct value +5